### ISSUE TITLE
inline-c-0.5.0.0 can't compile its test suite because of missing files

### ISSUE THREAD

**Issue by peti** [fpco/inline-c#15]
Citing from http://hydra.cryp.to/build/875479/nixlog/2/raw:

~~~~~~
In-place registering inline-c-0.5.0.0...
Preprocessing test suite 'tests' for inline-c-0.5.0.0...

test/tests.hs:15:18:
    Could not find module ‘Language.C.Inline.ContextSpec’
    Perhaps you meant
      Language.C.Inline.Context (from inline-c-0.5.0.0@inlin_9uLDEutUpjX8dfvvznZ5zI)
    Use -v to see a list of the files searched for.

test/tests.hs:16:18:
    Could not find module ‘Language.C.Inline.ParseSpec’
    Perhaps you meant
      Language.C.Inline.Unsafe (from inline-c-0.5.0.0@inlin_9uLDEutUpjX8dfvvznZ5zI)
    Use -v to see a list of the files searched for.

test/tests.hs:18:18:
    Could not find module ‘Language.C.Types.ParseSpec’
    Perhaps you meant
      Language.C.Types.Parse (from inline-c-0.5.0.0@inlin_9uLDEutUpjX8dfvvznZ5zI)
    Use -v to see a list of the files searched for.

test/tests.hs:20:18:
    Could not find module ‘Dummy’
    Use -v to see a list of the files searched for.
~~~~~~~

**Comment by bitonic** [comment]
Thanks, I forgot to include those modules in `other-modules`.  5777c8ac0b3135f741d6614c1be2196ef8a2f2e4 fixes this, I'll upload a new package soon.

**Comment by bitonic** [comment]
This is now fixed on hackage, `0.5.0.1`.
